packages/firebase_messaging/firebase_messaging/example/lib/permissions.dart

```diff
@@ -57,9 +57,9 @@ class _Permissions extends State<Permissions> {
     return Container(
       margin: const EdgeInsets.only(bottom: 8),
       child: Row(
-        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        // mainAxisAlignment: MainAxisAlignment.spaceBetween,
```

Comment:
Can you remove this line? I think it's better with Expanded 👍

Reply:
Thank you  I will delete this code.